packages/itinerary-body/src/stories/OtpRrItineraryBody.story.tsx

```diff
@@ -157,3 +160,57 @@ export const CustomTimeColumn = (): ReactElement => (
     TimeColumnContent={CustomTimeColumnContent}
   />
 );
+
+export const ThreeAlertsAlwaysCollapsing = (): ReactElement => (
+  <OtpRRItineraryBodyWrapper
+    itinerary={walkTransitWalkItinerary}
+    shouldAlwaysCollapseAlerts
+  />
+);
+
+export const TwoAlertsAlwaysCollapsing = (): ReactElement => (
+  <OtpRRItineraryBodyWrapper
+    itinerary={parkAndRideItinerary}
+    shouldAlwaysCollapseAlerts
+  />
+);
+
+export const ZeroAlertsAlwaysCollapsing = (): ReactElement => (
```

Comment:
I would think just one story with zero alerts is enough to demonstrate that no alert element is rendered at all. The other zero-alert cases seem redundant.

Reply:
While they give identical results, I wanted to easily test the logic of each. (If you'd prefer another way to do this, or think the tradeoff is not worthwhile, let me know and I'll happily update it.) For what it's worth, the "true param with zero alerts" helped me in the last commit - there was briefly a logic bomb in there that resulted in an error that was not present with a false param.